spyder/plugins/variableexplorer/widgets/dataframeeditor.py

```diff
@@ -45,10 +46,13 @@
     Signal, Slot)
 from qtpy.QtGui import QColor, QCursor
 from qtpy.QtWidgets import (
-    QApplication, QDialog, QFrame, QGridLayout, QHBoxLayout, QInputDialog,
-    QItemDelegate, QLabel, QLineEdit, QMessageBox, QPushButton, QScrollBar,
-    QStyle, QTableView, QTableWidget, QToolButton, QVBoxLayout, QWidget)
+    QApplication, QDialog, QFrame, QGridLayout, QHBoxLayout, QHeaderView,
+    QInputDialog, QItemDelegate, QLabel, QLineEdit, QMessageBox, QPushButton,
+    QScrollBar, QStyle, QTableView, QTableWidget, QToolButton, QVBoxLayout,
+    QWidget)
```

Comment:
```suggestion
    QApplication,
    QDialog,
    QFrame,
    QGridLayout,
    QHBoxLayout,
    QHeaderView,
    QInputDialog,
    QItemDelegate,
    QLabel,
    QLineEdit,
    QMessageBox,
    QPushButton,
    QScrollBar,
    QStyle,
    QTableView,
    QTableWidget,
    QToolButton,
    QVBoxLayout,
    QWidget,
)
```

Use Black formatting for this, which will simplify diffs and prevent conflicts in the future by having a single import per line.